css/style.scss

```diff
@@ -0,0 +1,382 @@
+@import "fonts";
+@import "slider";
+
+
+$mainBgColor: rgb(45, 48, 58);
+$mainBgColorBorder: rgb(50, 55, 70);
+$redColor: rgb(255, 108, 100);
+
+$navTextColor: rgb(255, 255, 255);
+$darkTextColor: rgb(102, 109, 137);
+
+$darkTextColorHover: rgb(142, 151, 189);
+
+
+html {
+      scroll-behavior: smooth;
+}
+body {
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+      box-sizing: border-box;
+}
+
+//                                   СТИЛИ HEADER
+.shadow {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 0%;
+      height: 0%;
+      z-index: 11;
+      background-color: rgba(45, 48, 58, 0);
+      transition: background-color 0.4s linear;
+}
+header {
+      position: fixed;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      border-bottom: 6px solid $mainBgColorBorder;
+      background-color: $mainBgColor;
+      height: 89px;
+      width: 100%;
+      z-index: 10;
+      top: 0;
+      left: 0;
+
+      section {
+            width: 1020px;
+            height: 100%;
+            display: flex;
+            flex-direction: row;
+      }
+}
+.burger {
+      display: none;
+}
+.header-article {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0px 40px 0px 40px;
+
+      h1 {
+            font-family: lato;
+            font-weight: bold;
+            font-size: 25px;
+            line-height: 30px;
+            letter-spacing: -0.03em;
+            color: $navTextColor;
+
+            span {
+                  color: $redColor;
+                  margin-left: 1px;
+            }
+      }
+
+      nav {
+            display: flex;
+            flex-direction: row;
+            width: 235px;
+            justify-content: space-between;
+      }
+}
+.nav__separator {
+      height: 2px;
+      width: 2px;
+      border-radius: 50%;
+      background-color: $navTextColor;
+      margin-top: 6px;
+}
+.nav-buttons {
+      font-family: lato;
+      font-weight: bold;
+      color: $navTextColor;
+      text-decoration: none;
+      font-size: 12px;
+      line-height: 14px;
+      letter-spacing: -0.035em;
+}
+.nav-buttons:hover {
+      color:$redColor
+}
+// .nav-buttons_active {
+//       color:$redColor
+// }
+.disabled {
+      pointer-events: none;
+}
+
+
+                                    // MAIN
+main {

```

Comment:
Don't use elements as a target for styles

Reply:
fixed